src/commands.py

```diff
@@ -257,6 +278,34 @@ def process_print_money(author: AccountId, message: str, server: Server, **kwarg
 
     return 'Money printed successfully.'
 
+
+def process_remove_funds(author: AccountId, message: str, server: Server, **kwargs):
+    author_account = assert_authorized(author, server, Authorization.ADMIN)
+    parsed = parse_remove_funds(message)
+    if parsed is None:
+        raise CommandException('Bad formatting; Expected `remove-funds` `AMOUNT` `BENEFICIARY`')
+    amount, beneficiary = parsed
+    if amount < 0:
+        raise CommandException('I can\'t remove negative amounts of funds')
+    beneficiary_account = assert_is_account(beneficiary, server)
+    if amount != 0:
+        server.remove_funds(author_account, beneficiary_account, amount)
+    return "Success"
+
+
+def parse_remove_funds(message):
```

Comment:
Isn't this the same as the parse function for print money?

Reply:
it is but calls a different server call